cn/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Distribute Credentials Securely Using Secrets
+---
+
+{% capture overview %}
+本文展示如何安全地将敏感数据（如密码和加密密钥）注入到 Pods 中。
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## 将 secret 数据转换为 base-64 形式
+
+假设用户想要有两条 secret 数据：用户名 `my-app` 和密码
+`39528$vdg7Jb`。 首先使用 [Base64 编码](https://www.base64encode.org/) 将用户名和密码转化为 base-64 形式。 这里是一个 Linux 示例：
+
+    echo -n 'my-app' | base64
+    echo -n '39528$vdg7Jb' | base64
+
+结果显示 base-64 形式的用户名为 `bXktYXBw`，
+base-64 形式的密码为 `Mzk1MjgkdmRnN0pi`。
+
+## 创建 Secret
+
+这里是一个配置文件，可以用来创建存有用户名和密码的 Secret:
+
+{% include code.html language="yaml" file="secret.yaml" ghlink="/docs/tasks/inject-data-application/secret.yaml" %}
```

Comment:
you may have to replicate the yaml file for the moment to make CI happy.
I hope this duplication issue can be solved in a decent way in future, then we can remove the duplicates.